test/suites/smoke/test-xcm-autopause.ts

```diff
@@ -0,0 +1,22 @@
+import "@moonbeam-network/api-augment/moonbase";
+import { describeSuite, expect } from "@moonwall/cli";
+
+describeSuite({
+  id: "S26",
+  title: `XCM Mode should be equal to Normal`,
+  foundationMethods: "read_only",
+  testCases: ({ context, it, log }) => {
+    it({
+      id: "C100",
+      title: "XCM Mode should be equal to Normal",
+      test: async function (context) {
+        if ((await context.polkadotJs().consts.system.version.specVersion) < 3200) {
+          context.skip();
```

Comment:
I think we need a return here, otherwise it may fail.
```suggestion
          return context.skip();
```

Reply:
Never mind, it seems that vitest handles this properly.

https://vitest.dev/guide/test-context.html#context-skip